biscuit-servant/README.md

```diff
@@ -1 +1,50 @@
-# Biscuit-based auth for servant apps
+<img src="https://raw.githubusercontent.com/divarvel/biscuit-haskell/main/assets/biscuit-logo.png" align=right>
+
+# biscuit-servant 🤖 [![Hackage][hackage]][hackage-url]
+
+> **Servant combinators to enable biscuit validation in your API trees**
+
+## Usage
+
+```Haskell
+type API = RequireBiscuit :> ProtectedAPI
+
+type ProtectedAPI =
+  "users" :> ( Get '[JSON] [User]
+             :<|> Capture "userId" Int :> Get '[JSON] User
+             )
+app :: PublicKey -> Application
+app pk = serveWithContext @API Proxy (genBiscuitCxt pk) server
+
+server :: Server API
+server b =
+  let handlers = userListHandler :<|> singleUserHandler
+      handleAuth =
+        handleBiscuit b
+        -- `allow if right(#authority, #admin);` will be the first policy
+        -- for every endpoint policies added by endpoints (or sub-apis) will
+        -- be appended.
+        . withPriorityVerifier [verifier|allow if right(#authority, #admin);|]
+        -- `deny if true;` will be the last policy for every endpoint
+        -- policies added by endpoints (or sub-apis) will be prepended.
+        . withFallbackVerifier [verifier|deny if true;|]
+  in hoistServer @ProtectedAPI Proxy handleAuth handlers
+
+allUsers :: [User]
+allUsers = [ User 1 "Danielle" "George"
+           , User 2 "Albert" "Einstein"
+           ]
+
+userListHandler :: HandlerAndVerifier [User]
```

Comment:
I think this is from the first draft of biscuit-servant and is outdated. `HandlerAndVerifier` does not exist anymore, it's `WithVerifier Handler` now